### ISSUE TITLE
Restart not support in Isolated mode

### ISSUE THREAD

**Issue by Harin8313** [Azure/azure-functions-durable-extension#2903]
### What version of .NET does your existing project use?

.NET 6

### What version of .NET are you attempting to target?

.NET 8

### Description

after migrating to isolated mode, restart feature is not work, is there any solution for this?
![image](https://github.com/user-attachments/assets/c97022b2-4686-410c-a272-f1320466e26d)

**Comment by andystaples** [comment]
Let's check if this is an issue with the HTTP Api returned by the orchestrator or something specific to the Azure Monitor. The former would be a much more serious issue

**Comment by andrewdmoreno** [comment]
It also does not exist on the `DurableTaskClient`. Any updates on this issue?

**Comment by renko7** [comment]
Can we please have analysis and guidance for this issue? Restarting failed orchestrations is extremely important for my application and is making he hesitate from moving to .NET 8 and the isolated model.

**Comment by cgillum** [comment]
I don't believe we have this API for .NET Isolated yet. @andystaples is this something we could put on your plate to look at?

Tagging @bachuv and @scale-tone since this is related to DFM. One thing that isn't clear to me yet is whether code changes will be needed in DFM (though I assume it may be necessary if the error message mentions .NET Isolated specifically).

**Comment by andystaples** [comment]
[like]  Andy Staples reacted to your message:
________________________________

**Comment by scale-tone** [comment]
> whether code changes will be needed in DFM

Yes, but just in form of adding a relevant method call [here](https://github.com/microsoft/DurableFunctionsMonitor/blob/cb09931de38b1be30cc189cf48c950c367f78446/durablefunctionsmonitor.dotnetisolated.core/Functions/Orchestration.cs#L204).
